cmake/up-core-api-protos.cmake

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2023 General Motors GTO LLC
+# Copyright (c) 2024 General Motors GTO LLC
```

Comment:
> Should we be updating copyright dates in files that haven't otherwise been changed? Copyrights are valid for _decades_ in most places, so doing a global find and replace in a repo just results in excess file churn.

I don't know why this was marked as resolved. Copyright date updates in files that were not otherwise modified should be reverted to minimize the number of unnecessary changes in this PR.